releasenotes/notes/datadog-fips-agent-builds-with-requirefips-59ce425386c3fd64.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    The `datadog-fips-agent` now builds with the ['requirefips' buildtag](https://github.com/microsoft/go/tree/microsoft/main/eng/doc/fips#build-option-to-require-fips-mode)
+    in the 7.64 linux release and no longer needs the `GOFIPS=1` environment
```

Comment:
```suggestion
    in the 7.64 Linux release and no longer needs the `GOFIPS=1` environment
```

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/33835/commits/e11185b2c1db1e0322dbb377a5f853d3702648e7